client/src/services/links.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const local = `http://localhost:4002`;
```

Comment:
can you put http://localhost:4002 in .env please (in client/.env, doesn't exist yet)